modules/rosa-required-aws-service-quotas.adoc

```diff
@@ -116,14 +122,14 @@ The default value of 5 vCPUs is not sufficient to create ROSA clusters. ROSA has
 |L-53DA6B97
 |50
 |50
-|
+|The maximum number of Application Load Balancers per Region
```

Comment:
Just checking that this needs to still be in title case in the description?

Reply:
Application Load Balancers does, to be distinct from Classic Load Balancers, but the rest doesn't - thanks!